receiver/prometheusreceiver/internal/metricfamily.go

```diff
@@ -297,7 +297,7 @@ func (mf *metricFamily) addSeries(seriesRef uint64, metricName string, ls labels
 			mg.complexValue = append(mg.complexValue, &dataPoint{value: v, boundary: boundary})
 		}
 	case pmetric.MetricTypeSum:
-		if strings.HasSuffix(metricName, metricSuffixCreated) {
+		if strings.HasSuffix(metricName, metricSuffixCreated) && metricName != mf.metadata.Metric {
```

Comment:
```suggestion
		if metricName == mf.metadata.Metric + "_created" {
```
Should we check that it is specifically the metric name + suffix?

Reply:
I am okay with that but in case the metadata is missing, the behaviour will now change, even if we are not sure if it's _created timestamp or just metric with the name as suffix.

The current implementation preserves the old behaviour unless we know from metadata that it's not _created timestamp

Reply:
Do we know in what scenarios metatada is missing?  Since `_created` timestamp is new, I'm assuming only series that have metadata would every have it.  Maybe that isn't a good asumption?

Reply:
I have seen people not using the official client libraries or any for that matter and just using basically prints to output metrics, I can imagine the metadata missing in such cases (not really sure how common it is), but I guess in those case the timestamp wouldn't be there either.